test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2080,6 +2193,352 @@ def test_cgroup_use_hierarchy(self):
         else:
             self.assertFalse(1, "File %s not present" % fpath)
 
+    def check_req_rjs(self):
+        """
+        Check the requirements for the reliable job startup tests.
+        Return 1 if requirements are not satisfied.
+        """
+        ret = 0
+        # Check that momA has at least two free vnodes
+        attr = {'state': 'free'}
+        rv1 = True
+        try:
+            self.server.expect(VNODE, attr, id='%s[0]' % self.hostA,
+                               max_attempts=3, interval=2)
+        except PtlExpectError, e:
```

Comment:
Please use "except PtlExceptError as exc:"
There are multiple cases of this.

Reply:
Done now in multiple places.